internal/revision/revision.go

```diff
@@ -186,24 +186,28 @@ func buildTokenBufer(previous *pb.RevisionTokenData, eKeys []*model.Exposure) *p
 		RevisableKeys: make([]*pb.RevisableKey, 0, len(eKeys)),
 	}
 	got := make(map[string]struct{})
-	for _, k := range eKeys {
-		got[k.ExposureKeyBase64()] = struct{}{}
-		pbKey := pb.RevisableKey{
-			TemporaryExposureKey: make([]byte, len(k.ExposureKey)),
-			IntervalNumber:       k.IntervalNumber,
-			IntervalCount:        k.IntervalCount,
-		}
-		copy(pbKey.TemporaryExposureKey, k.ExposureKey)
-		tokenData.RevisableKeys = append(tokenData.RevisableKeys, &pbKey)
-	}
-	// Add in previous keys that weren't also in the new exposures
+
+	// Add in previous keys that weren't also in the new exposures. This needs to
```

Comment:
comment is wrong, it’s just all previous keys from the previous revision token.

Reply:
DOne